coralproject/talk
# [CORL-1361] Add badge class names to root of comments

## What does this PR do?

If we have badges, use them to set a classname so we can bind to it.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- edit badges on a user
- post a comment as the user
- check the DOM and see that `badge-<your-badge>` is set on the root div for the comment
